src/Orleans/Streams/PubSub/ImplicitStreamSubscriberTable.cs

```diff
@@ -8,11 +10,13 @@ namespace Orleans.Streams
     [Serializable]
     internal class ImplicitStreamSubscriberTable
     {
-        private readonly Dictionary<string, HashSet<int>> table;
+        private readonly ConcurrentDictionary<string, HashSet<int>> table;
```

Comment:
Table is now just a cache, and is dynamically updated at runtime, it no longer should be serialized as part of the ImplicitStreamSubscriberTable.
The predicates are ultimately the authority and the only data that needs serialized.
Or am I missing something?